eZ/Publish/Core/Persistence/Legacy/Content/Type/Gateway.php

```diff
@@ -294,10 +294,10 @@
     /**
      * Returns field mapping data
      *
-     * Returns an associative array with ContentType and FieldDefinition identifiers as
-     * first and second level keys respectively, and FieldDefinition ID as value.
+     * Returns an associative array with ContentType and searchable FieldDefinition identifiers as
+     * first and second level keys respectively, and FieldType identifier as value.
      *
      * @return array
      */
-    abstract public function getFieldMap();
+    abstract public function getFieldMapData();
```

Comment:
Changed in https://github.com/ezsystems/ezpublish-kernel/commit/2570eed21b9f101b9324116957bac705af575338.